### PR TITLE
further doc expansion

### PR DESCRIPTION
**Pull request by SethTisue**
review by @adriaanm plz

### CODE REVIEW

File: doc/maintenance.md [hunk 1]
```diff
@@ -1,17 +1,95 @@
...
+The commands to run chef-client depend on whether the node in question
+is Linux or Windows:
+
+#### Linux node
+
+Here hostname might be e.g. `jenkins-master` and username could be omitted
+to accept the default in your `~/.ssh/config`, or might be e.g. `ec2-user`.
+
+```
+ssh username@hostname
```
Comment (adriaanm) [inline doc/maintenance.md]: if you include the hostname in the config file as suggested somewhere else, it's just `ssh jenkins-master`

**Comment by adriaanm** [comment]
Also, it turns out the validation key is not needed for `knife cookbook upload`, since you could upload without it.

**Comment by SethTisue** [comment]
re: the validation key, so then I can remove this text from `client-setup.md`:

> Obtain the organization validation key from Adriaan and put it to `$PWD/.chef/config/$CHEF_ORG-validator.pem`. (Q: When is this key used exactly? https://docs.chef.io/chef_private_keys.html says it's when a new node runs `chef-client` for the first time.)

But where do I put it instead?

File: doc/maintenance.md (hunk 1 above)
Comment (SethTisue) [inline doc/maintenance.md]: addressed in additional commit

**Comment by adriaanm** [comment]
genesis?

**Pull request merged**